backend/api/views/TimeRecordView.py

```diff
@@ -0,0 +1,83 @@
+from rest_framework import viewsets
+from rest_framework.permissions import IsAuthenticated
+from api.models import TimeRecord, EventType, Group
+from api.serializers.TimeRecordSerializer import TimeRecordSerializer
+from drf_spectacular.utils import extend_schema, OpenApiParameter
+from django.db.models import Q, F, Value, Func, IntegerField
+from django.db.models.functions import Collate, Concat
+from rest_framework.exceptions import ValidationError
+from rest_framework import status
+from django.utils import timezone
+
+import logging
+
+logger = logging.getLogger('django')
+
+
+@extend_schema(tags=['Time Record'], request=TimeRecordSerializer)
+class TimeRecordViewSet(viewsets.ModelViewSet):
+    queryset = TimeRecord.objects.all()
+    serializer_class = TimeRecordSerializer
+    permission_classes = [IsAuthenticated]
+    http_method_names = ['get', 'post', 'delete', 'patch']
```

Comment:
Right now we are ordering by id... I was thinking that maybe it is a good idea to order by first name. What do you think?

Reply:
Sounds good. I just added the order by athlete first name by default.